hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/TarContainerPacker.java

```diff
@@ -56,6 +57,8 @@
 
   static final String DB_DIR_NAME = "db";
 
+  static final String DUMP_DIR_NAME = "dump";
```

Comment:
Can we leverage the "db" directory name, since the dumped files are true db files, and "db" is more straightforward?

Reply:
I think the idea is good, but actually the handling is different for original rocksdb files and the dumped sst files, I'll explain it in the comment below.